math/blis/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           github 1.0
+
+github.setup        flame blis 0.9.0
+revision            0
+categories          math
+maintainers         {@barracuda156 gmail.com:vital.had} {@catap korins.ky:kirill} openmaintainer
+license             BSD
+description         BLAS-like Library Instantiation Software Framework
+long_description    BLIS is an award-winning portable software framework for instantiating \
+                    high-performance BLAS-like dense linear algebra libraries. The framework \
+                    was designed to isolate essential kernels of computation that, when optimized, \
+                    immediately enable optimized implementations of most of its commonly used \
+                    and computationally intensive operations.
+github.tarball_from archive
+checksums           rmd160  6661deb6d059babb6688ad274db09286482f6202 \
+                    sha256  1135f664be7355427b91025075562805cdc6cc730d3173f83533b2c5dcc2f308 \
+                    size    15078619
+
+compiler.c_standard 1999
+
+compiler.blacklist-append \
+                    *gcc-4.* {clang < 500}
+
+configure.args-append \
+                    --enable-blas \
+                    --enable-threading=pthread \
+                    --enable-system \
+                    --prefix=${prefix} \
```

Comment:
No need to put `--prefix=${prefix}` in `configure.args`; it's already in `configure.pre_args` by default.

Reply:
©ryandesign Thank you. Should we remove it now or later together with a future update of the port?

On a side note, how to clear select `pre_args`? (I actually need `--prefix=` not to be passed at that point for a specific case, but not sure if clearing all `pre_args` is the only option.)

Reply:
> On a side note, how to clear select `pre_args`?

Just use `configure.pre_args` :)

Reply:
or “configure.pre_args-delete XYZ” to be selective.

Reply:
> > On a side note, how to clear select `pre_args`?
> 
> Just use `configure.pre_args` :)

@catap That was obvious, yes :)

@kencu Thanks!